sample/OSCON17/sponsors

```diff
@@ -0,0 +1,362 @@
+        "description": ""
+    }, 
+    {
+        "id": 7, 
+        "name": "PayPal", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFaDJsZVRmNDNCc2M", 
+        "level": "4", 
+        "sponsor_type": "Rhodium", 
+        "url": "https://paypal.github.io/", 
+        "description": ""
+    }, 
+    {
+        "id": 8, 
+        "name": "Akamai Developer", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFemNEY2hXTnZuY1k", 
+        "level": "4", 
+        "sponsor_type": "Rhodium", 
+        "url": "http://developer.akamai.com/", 
+        "description": ""
+    }, 
+    {
+        "id": 9, 
+        "name": "Capital one", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFbGJXUWVyVWFNVGc", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "http://www.capitalone.com/", 
+        "description": ""
+    }, 
+    {
+        "id": 10, 
+        "name": "Intel", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFdkJwbUhjTnFycVU", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "http://www.intel.com/opensource", 
+        "description": ""
+    },
+    {
+        "id": 11, 
+        "name": "Microsoft", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFV1F6OWZPbWdydFU", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "https://conferences.oreilly.com/oscon/opensource.microsoft.com", 
+        "description": ""
+    },
+    {
+        "id": 12, 
+        "name": "Pivotal", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFQ1hseUcyaDlwbUE", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "http://pivotal.io/", 
+        "description": ""
+    }, 
+    {
+        "id": 13, 
+        "name": "RedHat", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFdnNzNks1amh5ajA", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "http://www.redhat.com/", 
+        "description": ""
+    }, 
+    {
+        "id": 14, 
+        "name": "salesforce", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFOElsSWVrb080aTA", 
+        "level": "5", 
+        "sponsor_type": "Gold", 
+        "url": "http://www.salesforce.com/tech", 
+        "description": ""
+    },
+     {
+        "id": 15, 
+        "name": "CloudFoundry", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFai1VbnhNb0s5V1U", 
+        "level": "6", 
+        "sponsor_type": "Silver", 
+        "url": "http://www.cloudfoundry.org/", 
+        "description": ""
+     },
+     {
+        "id": 16, 
+        "name": "Conde Nast", 
+        "logo": "https://drive.google.com/uc?export=download&id=0B2b0W4GqpgLFM2pBSzc3bVdQSFE", 
+        "level": "6", 
+        "sponsor_type": "Silver", 
+        "url": null, 
```

Comment:
The URL given on the site for this sponsor was invalid so I decided to not mention it here @Shailesh351